README.md

```diff
@@ -1,8 +1,78 @@
-# rclnodejs
+# rclnodejs - ROS Client Library for JavaScript language
 
-ROS2.0 Client Library for JavaScript with nodejs
+Branch | Linux Build | MacOS Build | Windows Build |
+------------ | :-------------: |  :-------------: |  :-------------: |
+develop | [![Build Status](https://travis-ci.org/RobotWebTools/rclnodejs.svg?branch=develop)](https://github.com/RobotWebTools/rclnodejs/tree/develop) | N/A | N/A
+master | [![Build Status](https://travis-ci.org/RobotWebTools/rclnodejs.svg?branch=master)](https://github.com/RobotWebTools/rclnodejs/tree/master) | N/A | N/A
 
-Branch | Build Status |
------------- | :-------------: |
-develop | [![Build Status](https://travis-ci.org/RobotWebTools/rclnodejs.svg?branch=develop)](https://github.com/RobotWebTools/rclnodejs/tree/develop)
-master | [![Build Status](https://travis-ci.org/RobotWebTools/rclnodejs.svg?branch=master)](https://github.com/RobotWebTools/rclnodejs/tree/master)
+## Build Environment
+
+1.Building ROS 2.
+
+Currently, the ROS is a cross-platform system, which covers Linux, MacOS and Windows, and the `rclnodejs` module is developed on the [`master`](https://github.com/ros2/ros2/blob/master/ros2.repos) branch of ROS, so you have to build ROS from source at the present stage. Please select the platform you want to work on, then reference the instruction of building ROS.
+
+* [Linux](https://github.com/ros2/ros2/wiki/Linux-Development-Setup)
+* [MacOS](https://github.com/ros2/ros2/wiki/OSX-Development-Setup)
+* [Windows](https://github.com/ros2/ros2/wiki/Windows-Development-Setup)
+
+2.Install `Node.js`.
+
+The `Node.js` version we selected is the latest LTS `Boron`. You can install it:
+
+* Download from Node.js offical [website](https://nodejs.org/en/), and install it.
+* Use the Node Version Manager ([nvm](https://github.com/creationix/nvm)) to install it.
+
+3.Install `node-gyp`, the build tool for `rclnodejs`.
+
+```javascript
+npm install -g node-gyp
+```
+
+## Build Module
+
+This `Node.js` module is built by [node-gyp](https://www.npmjs.com/package/node-gyp), all you have to do is just to run the following command (**We only support the Linux platform in the current phase**):
+
+```javascript
```

Comment:
The documentation above is telling users to build ROS2 from source, and it's with `--symlink-install` parameter by default. In that case, users should run `source install/local_setup.bash` first.

Reply:
That' right, thanks.